### PR TITLE
admin-user-controller-test

### PR DESCRIPTION
**Pull request by ngovannghiaframgia**
Screenshot from 2019-03-12 16-25-28

### CODE REVIEW

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by ngovannghiaframgia** [comment]
ready

File: nano.save [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (huongnguyenmta) [inline nano.save]: file gì đây em??

**Comment by ngovannghiaframgia** [comment]
ready

**Pull request merged**